### PR TITLE
feat: Add .gitignore with .DS_Store. package-lock.json yarn-error.log

### PR DESCRIPTION
**Pull request by DonDonDonDonDonDon**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 .vscode
 .idea
-/node_modules
\ No newline at end of file
+/node_modules
+.DS_Store
+package-lock.json
```
Comment (vasilev-alex) [inline .gitignore:5]: `package-lock.json` should be committed. But yeah, we need to add the rest

**Comment by vasilev-alex** [comment]
Hey! Thanks for this one.

However I moved `templates-core` to `packages` in the meanwhile :) Could you please resolve the conflict and we're ready to go

**Comment by DonDonDonDonDonDon** [comment]
😃发自我的iPhone------------------ Original ------------------From: Alex.V  <notifications@github.com>Date: Fri,Jul 31,2020 10:24 PMTo: cube-js/cubejs-playground-templates <cubejs-playground-templates@noreply.github.com>Cc: DonDonDonDonDonDon <18353150234@163.com>, Author <author@noreply.github.com>Subject: Re: [cube-js/cubejs-playground-templates] feat: Add .gitignore with.DS_Store. package-lock.json yarn-error.log (#1)
@vasilev-alex commented on this pull request.



In .gitignore:
> \ No newline at end of file
+/node_modules
+.DS_Store
+package-lock.json

package-lock.json should be committed. But yeah, we need to add the rest

—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/cube-js/cubejs-playground-templates/pull/1#pullrequestreview-459214530",
"url": "https://github.com/cube-js/cubejs-playground-templates/pull/1#pullrequestreview-459214530",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

**Comment by DonDonDonDonDonDon** [comment]
> Hey! Thanks for this one.
> 
> However I moved `templates-core` to `packages` in the meanwhile :) Could you please resolve the conflict and we're ready to go

I want to do more for cubejs-playground-templates,but i don't know how to use it?
 I downloaded yesterday,but not work.Where do I start?

**Pull request closed without merging**